### DISCUSSION TITLE
[C] prefetch hint for the loops

### DISCUSSION THREAD

**Discussion by Slapparoo** [PlummersSoftwareLLC/Primes#435] [Improvement suggestions]
I have a variant of the C solution 1, which I have included cache prefetch hints for the loops which offers a marginal gain (just ~ 1%) I was wondering if this is in the spirit? or if its a little try hard.

**Comment by rbergen** [comment]
@Slapparoo I for one never make up my mind about an idea until I have source code to look at. 

In a case where the performance improvement is this small, you may want to tag the original author of the solution you're aiming to improve, and ask them to comment.
